js/lib/Country.js

```diff
@@ -108,45 +108,45 @@ var Country = function (options) {
 /**
  * Return an array of the ids for all ISO 3166-1 alpha-2 code that
  * this copy of ilib knows about.
- * 
+ *
  * @static
  * @return {Object} an object of country code that this copy of ilib knows about.
  */
 Country.getAvailableCode = function() {
 	var ret = [],
-		country,
+		code,
 		countries = new ResBundle({
-			name: "ctrynames"
+			name: "ctryreverse"
 		}).getResObj();
-	
-	for (country in countries) {
-		if (country && countries[country]) {
-			ret.push(countries[country]);
+
+	for (code in countries) {
+		if (code && countries[code]) {
+			ret.push(code);
 		}
-	}		
-	
+	}
+
 	return ret;
 };
 
 /**
  * Return an array of country names that this copy of ilib knows about.
- * 
+ *
  * @static
  * @return {Object} an object of country code that this copy of ilib knows about.
  */
 Country.getAvailableCountry = function() {
 	var ret = [],
-		country,
+		code,
 		countries = new ResBundle({
-			name: "ctrynames"
+			name: "ctryreverse"
 		}).getResObj();
-	
-	for (country in countries) {
-		if (country && countries[country]) {
-			ret.push(country);
+
+	for (code in countries) {
+		if (code && countries[code]) {
+			ret.push(countries[code]);
```

Comment:
There should be a private method that calculates the reverse (it sets this.countryToCode) which should be called here. Then, this function would be just like the other one above:

```
if (!this.countryToCode) {
  this._calculateCountryToCode();
}
return Object.keys(this.countryToCode);
```

Reply:
I have a question.. `getAvailableCountry` is static method. Is it possible to call private method in static method(when `Country` instance does not exist)?